### PR TITLE
Fix AbstractCaseInstancePresenter place req.param validation

### PR DESCRIPTION
**Pull request by jhrcek**
@cristianonicolai Validation change we discussed. But when building jbpm-console-ng locally, i'm getting 1 reproducible fail in `CaseRolesPresenterTest.testLoadCaseInstance` but I have no idea how this is related to my change..

### CODE REVIEW

File: jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/util/AbstractCaseInstancePresenter.java [hunk 1]
```diff
@@ -69,9 +69,9 @@ public void findCaseInstance() {
     }
 
     protected boolean isCaseInstanceValid() {
-        return isNullOrEmpty(serverTemplateId) == false ||
-               isNullOrEmpty(containerId) == false ||
-               isNullOrEmpty(caseId)  == false;
+        return isNullOrEmpty(serverTemplateId) == false
```
Comment (cristianonicolai) [inline jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/util/AbstractCaseInstancePresenter.java]: serverTemplateId is not actually in use at the moment, fixing the validation would mean that you have to exclude it from the check. Keeping only containerId and caseId

File: jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/test/java/org/jbpm/console/ng/cm/client/comments/CaseCommentsPresenterTest.java [hunk 2]
```diff
@@ -147,5 +152,39 @@ public void testDeleteComment() {
...
+                verifyGetCaseInstanceCalled(invalidParams, 0)
+        );
+        // Only all NOT-null constitute valid place request params
+        verifyGetCaseInstanceCalled(new String[]{"serverTemplateId", "containerId", "caseId"}, 1);
+    }
 
+    private void verifyGetCaseInstanceCalled(String[] placeRequestParams, int timesCalled) {
+        PlaceRequest placeRequest = new DefaultPlaceRequest(
+                CaseCommentsPresenter.SCREEN_ID,
+                new HashMap<String, String>() {{
```
Comment (cristianonicolai) [inline jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/test/java/org/jbpm/console/ng/cm/client/comments/CaseCommentsPresenterTest.java]: Could we avoid using this inline instantiation style?

Comment (jhrcek) [inline jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/test/java/org/jbpm/console/ng/cm/client/comments/CaseCommentsPresenterTest.java]: Sure, I'll update the PR

**Comment by cristianonicolai** [comment]
@jhrcek reason the CaseRoles test is failing is due to the lack of containerId and caseId parameters at the CaseInstanceSummary object. Something like this should do:
`final CaseInstanceSummary cis = CaseInstanceSummary.builder().containerId("containerId").caseId("caseId").roleAssignments(singletonList(cras)).build();
`

**Comment by cristianonicolai** [comment]
@jhrcek, I made some extra comments in your PR, not sure if you had a chance to have a look.

**Comment by jhrcek** [comment]
@cristianonicolai thanks, updated based on your comments.

**Comment by jhrcek** [comment]
Fixed merge conflict

**Pull request merged**